src/tools/compiletest/src/runtest.rs

```diff
@@ -1420,6 +1457,11 @@ impl<'test> TestCx<'test> {
             rustc.arg(dir_opt);
         };
 
+        // `use-minicore` requires `#![no_std]` and `#![no_core]`, which means no unwinding panics.
+        if self.props.use_minicore {
+            rustc.arg("-Cpanic=abort");
+        }
```

Comment:
Note: I need to document `//@ use-minicore` implies and mandates `-C panic=abort` for the test in rustc-dev-guide.

Reply:
https://github.com/rust-lang/rustc-dev-guide/pull/2097